### ISSUE TITLE
Quick panning the map causes view.getCenter() to report [NaN, NaN]

### ISSUE THREAD

**Issue by fargraph** [openlayers/openlayers#14426]
**Describe the bug**
In our application we disable kinetic panning, but when you quickly and repeatedly grab the map to pan it, the center starts reporting that it is `[NaN, NaN]`

You can see in this screenshot, I attached a `change:center` listener to the view and then proceeded to quickly and repeatedly pan the map.  The output looks like this:
![image](https://user-images.githubusercontent.com/17570707/210849287-bc761bac-0567-4947-aaa5-3792fe836688.png)

Once this occurs, the map is in an error state and a refresh is required.

If you just grab the map once and pan around whithout letting the mouse button go, it does not cause the error.

**To Reproduce**
1. Load a map that has the `DragPan` interaction intialized with `{kinetic: Kinetic(0,0,0)}` as its constructor property. See this [code sandbox](https://codesandbox.io/s/quick-panning-causes-nan-9c4zch?file=/main.js) for a demo of the bug
2. click and drag the map quickly and repeatedly.
3. eventually the `ol/View` object will start reporting the center as `[NaN, NaN]`

**Expected behavior**
I would expect the center to be a correct number.

**Comment by mike-000** [comment]
The docs specify the kinetic `decay` must be negative.  A zero value will cause a zero divide when calculating distance travelled.  Try using a very small negative value such as `-1e-10`.

**Comment by fargraph** [comment]
You are correct, this fixes the problem and does not seem to have the same effect.  Thanks!

**Comment by tschaub** [comment]
@fargraph - have you tried creating a `DragPan` interaction with no `kinetic` option?

**Comment by fargraph** [comment]
> @fargraph - have you tried creating a `DragPan` interaction with no `kinetic` option?

Yeah, and that works fine.  At some point I wanted to disable kinetic scrolling, probably so I wouldn't need to debounce my fetches to populate the map as heavily.  I saw a stack exchange article describing how to turn off the kinetic panning using `Kinetic(0,0,0)`.  It worked without throwing an error or warning in the console, so I didn't think it was necessary to check the docs.

Whatever my reason before, it doesn't seem to still be an issue for me, so I decided to allow the default kinetic behavior.  However, if I wanted to disable the kinetic behavior,   @mike-000's proposed solution works too.

I do think some defensive coding would be helpful, for a condition like "must be a negative number" but that's up to the library maintainers whether or not they want to implement that.  If there's a long standing convention to not check for valid arguments, then I suppose this issue can be closed.

**Comment by nboisteault** [comment]
@fargraph Thanks for raising this issue. I have added a comment to the [stack overflow answer](https://stackoverflow.com/a/65307106/2000654) as I can't edit it due to too much pending edits.

**Comment by ahocevar** [comment]
Thanks @nboisteault, I updated the Stack Overflow answer.
